rust/arrow/src/compute/kernels/arithmetic.rs

```diff
@@ -170,7 +225,7 @@ where
     // Create the combined `Bitmap`
     let null_bit_buffer =
         combine_option_bitmap(left.data_ref(), right.data_ref(), left.len())?;
-    let bitmap = null_bit_buffer.map(Bitmap::from);
+    let bitmap = null_bit_buffer.clone().map(Bitmap::from);
```

Comment:
Is this clone necessary?

Reply:
Maybe not, but I was unable to get a bitmap reference to set the mask for SIMD from a buffer without `clone`.